### PR TITLE
Swapsicle - Add Taiko

### PR DESCRIPTION
**Pull request by Adonis2115**

### CODE REVIEW

**Comment by llamatester** [comment]
The swapsicle-v2 adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts dexs swapsicle-v2
    
    🦙 Running SWAPSICLE-V2 adapter 🦙
    ---------------------------------------------------
    Start Date:	Thu, 17 Oct 2024 00:00:00 GMT
    End Date:	Fri, 18 Oct 2024 00:00:00 GMT
    ---------------------------------------------------
    
    MANTLE 👇
    Backfill start time: 13/10/2023
    NO METHODOLOGY SPECIFIED
    End timestamp: 1729209600 (2024-10-18T00:00:00.000Z)
    Block: 70.54 M
    Total volume: 10.33 M
    Daily volume: 9.41 k
    
    
    TELOS 👇
    Backfill start time: 24/10/2023
    NO METHODOLOGY SPECIFIED
    End timestamp: 1729209600 (2024-10-18T00:00:00.000Z)
    Block: 368.47 M
    Total volume: 63.29 M
    Daily volume: 636.26 k
    
    
    TAIKO 👇
    Backfill start time: 4/1/1970
    NO METHODOLOGY SPECIFIED
    End timestamp: 1729209600 (2024-10-18T00:00:00.000Z)
    Block: 467.71 k
    Total volume: 46.89 k
    Daily volume: 1.67 k

File: dexs/swapsicle-v2/index.ts [hunk 1]
```diff
@@ -13,4 +14,5 @@ const adapters = univ2Adapter({
 
 adapters.adapter[CHAIN.MANTLE].start = 1697155200;
 adapters.adapter[CHAIN.TELOS].start = 1698105600;
+adapters.adapter[CHAIN.TAIKO].start = 338445;
```
Comment (g1nt0ki) [inline dexs/swapsicle-v2/index.ts]: change this to unix timestamp

Comment (Adonis2115) [reply dexs/swapsicle-v2/index.ts]: Resolved, Thanks

**Comment by g1nt0ki** [comment]
@Adonis2115 thanks for the update

**Comment by llamatester** [comment]
The swapsicle-v2 adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts dexs swapsicle-v2
    
    🦙 Running SWAPSICLE-V2 adapter 🦙
    ---------------------------------------------------
    Start Date:	Thu, 17 Oct 2024 00:00:00 GMT
    End Date:	Fri, 18 Oct 2024 00:00:00 GMT
    ---------------------------------------------------
    
    MANTLE 👇
    Backfill start time: 13/10/2023
    NO METHODOLOGY SPECIFIED
    End timestamp: 1729209600 (2024-10-18T00:00:00.000Z)
    Block: 70.54 M
    Total volume: 10.33 M
    Daily volume: 9.41 k
    
    
    TELOS 👇
    Backfill start time: 24/10/2023
    NO METHODOLOGY SPECIFIED
    End timestamp: 1729209600 (2024-10-18T00:00:00.000Z)
    Block: 368.47 M
    Total volume: 63.29 M
    Daily volume: 636.26 k
    
    
    TAIKO 👇
    Backfill start time: 29/8/2024
    NO METHODOLOGY SPECIFIED
    End timestamp: 1729209600 (2024-10-18T00:00:00.000Z)
    Block: 467.71 k
    Total volume: 46.89 k
    Daily volume: 1.67 k

**Pull request merged**